Tools/LyTestTools/ly_test_tools/o3de/multi_test_framework.py

```diff
@@ -787,9 +787,9 @@ def _exec_single_test(self,
                        f"-pythontestcase={test_case_name}",
                        "-logfile", f"@log@/{log_name}",
                        "-project-log-path", log_path_function(run_id, workspace)] + test_cmdline_args
-        elif type(executable) in [WinMaterialEditor, LinuxMaterialEditor]:
-            log_path_function = editor_utils.retrieve_material_editor_log_path
-            log_content_function = editor_utils.retrieve_material_editor_log_content
+        elif type(executable) in [WinMaterialEditor, LinuxMaterialEditor, WinMaterialCanvas, LinuxMaterialCanvas]:
+            log_path_function = editor_utils.retrieve_non_editor_log_path
+            log_content_function = editor_utils.retrieve_non_editor_log_content
```

Comment:
Adapting to every possible "non_editor_log" seems like an unreasonably wide scope to cover, so I suspect this is misnamed. Perhaps this should be something like 'launcher_log' or 'execution_log' or 'tool_log'?

Reply:
actually the scope has widened for anything non-editor - see @gadams3 post about the combined atomtools functionality

Reply:
having said that, I can try to find a better name for it

Reply:
Starting to sound like an 'atomtools_log'